src/auth/controllers/auth.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Post, UseGuards, Request } from '@nestjs/common';
+import { AuthService } from '../services/auth.service';
+import { CustomersService } from 'src/customers/service/customers.service';
+
+import { LocalAuthGuard } from '../guards/local-auth.guard';
+import { RefreshJwtGuard } from '../guards/refresh-jwt-auth.guards';
+
+@Controller('auth')
+export class AuthController {
+  constructor(
+    private authService: AuthService,
+    private customersService: CustomersService,
+  ) {}
+
+  @UseGuards(LocalAuthGuard)
+  @Post('logIn')
```

Comment:
Check this: [REST API Naming Conventions and Best Practices](https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5)

Meaning: `log-in` is preferred.